File: kotlin-ide/j2k/new/testData/newJ2k/methodCallExpression/vararg2.java
//file
import javaApi.WithVarargConstructor;

import java.lang.String;

class X {
    void foo() {
        WithVarargConstructor o1 = new WithVarargConstructor(1, new Object[]{"a"});
        WithVarargConstructor o2 = new WithVarargConstructor(2, new Object[]{"a"}, new Object[]{"b"});
        WithVarargConstructor o3 = new WithVarargConstructor(2, "a");
    }
}

File: j2k/testData/fileOrElement/methodCallExpression/vararg2.kt
import javaApi.WithVarargConstructor

internal class X {
    fun foo() {
        val o1 = WithVarargConstructor(1, *arrayOf<Any>("a"))
        val o2 = WithVarargConstructor(2, arrayOf<Any>("a"), arrayOf<Any>("b"))
        val o3 = WithVarargConstructor(2, "a")
    }
}